src/java.base/share/classes/sun/security/ssl/BaseSSLSocketImpl.java

```diff
@@ -262,37 +262,6 @@ public boolean isOutputShutdown() {
         }
     }
 
-    /**
```

Comment:
Can you please add a quick couple lines here with the technical rationale for removing it, so we don't forget down the road.

i.e.

    There used to be a finalize() here, but decided that was not
    needed.  If users don't properly close the socket...

    The native resources are handled by the Socket Cleaner.

Reply:
It may be not common to comment on non-existing method.  Maybe,  the class description could be a place to add this note.